plugin/instance/vagrant/instance.go

```diff
@@ -74,6 +73,10 @@ func (v vagrantPlugin) Provision(spec instance.Spec) (*instance.ID, error) {
 	}
 
 	templ := template.Must(template.New("").Parse(vagrantFile))
+	if _, err := os.Stat(v.VagrantfilesDir + "vagrant.tmpl"); err == nil {
```

Comment:
I'm not a big fan of coupling the template with the data dir.  For example, it's nice if the dir could be considered disposable (but not the template).  How about a `--template` argument to the plugin instead?